Add basic documentation (#21)
qutip/qutip-jax: doc/source/solver.rst

.. _qtjax_solver:

*****************************

> Jax BackEnd in Qutip's Solver

Reviewer: ```suggestion
Jax Backend in Qutip's Solver
```

*****************************

.. _mesolve:

Using Jax in ``mesolve``
========================

> To use the jax arrays in Qutip solver such as ``mesolve``, two conditions must be fullfill:

Reviewer: ```suggestion
To use JAX's arrays in QuTiP's solvers such as ``mesolve``, two conditions must be met:
```

> 1. All opertors and states must use jax arrays.

Reviewer: ```suggestion
1. All opertors and states must use JAX arrays.
```

> 2. An ODE integrator supporting Jax must be used.
>    ODE solver from the diffrax project are made available when importing qutip-jax

Reviewer: ```suggestion
2. An ODE integrator supporting Jax must be used.
   Currently, an ODE solver from the diffrax project is made available when importing qutip-jax.
   
The following code shows an example of how to use JAX:
```

I remember you mentioned that JAX  was a bit slower the first time it is executed. Is this always the case? If so, perhaps we should briefly explain here why this happens.

Author: The first time compile the functions. It's true for any `jit` function, but the compilation is quite quick.

.. _qtjax_solver:

*****************************

> Jax Backend in Qutip's Solver

Reviewer: ```suggestion
Jax Backend in QuTiP's Solver
```
